pandas/core/frame.py

```diff
@@ -8394,6 +8394,52 @@ def isin(self, values):
                 self.columns,
             )
 
+    def value_counts(self):
+        """
+        The number of times each unique row appears in the DataFrame.
+
+        Rows that contain any NaN value are omitted from the results.
+
+        Returns
+        -------
+        counts : Series
+
+        See Also
+        --------
+        Series.value_counts: Equivalent method on Series.
```

Comment:
we have more options on the Series.value_counts, dropna for example these need to be implemented